mgmt_cli_test.py

```diff
@@ -1677,52 +1678,10 @@ def create_backup_and_report(self, mgr_cluster, label: str):
         self.report_to_argus(ManagerReportType.BACKUP, backup_report, label)
         return task
 
-    def run_read_stress_and_report(self, label):
-        stress_queue = {"read": [], "write": []}
-
-        for command in self.params.get('stress_cmd'):
-            if " write " in command:
-                stress_queue["write"].append(self.run_stress_thread(command))
-            elif " read " in command:
-                stress_queue["read"].append(self.run_stress_thread(command))
-            else:
-                raise ValueError("Unknown stress command")
-
-        def get_stress_averages(queue):
-            averages = {'op rate': 0.0, 'partition rate': 0.0, 'row rate': 0.0, 'latency 99th percentile': 0.0}
-            num_results = 0
-            for stress in queue:
-                results = self.get_stress_results(queue=stress)
-                num_results += len(results)
-                for result in results:
-                    for key in averages:
-                        averages[key] += float(result[key])
-            stats = {key: averages[key] / num_results for key in averages}
-            return stats
-
-        with ExecutionTimer() as stress_timer:
-            read_stats = get_stress_averages(stress_queue["read"])
-            write_stats = get_stress_averages(stress_queue["write"])
-
-        InfoEvent(message=f'Read stress duration: {stress_timer.duration}s.').publish()
-
-        read_stress_report = {
-            "read time": int(stress_timer.duration.total_seconds()),
-            "op rate": read_stats['op rate'],
-            "partition rate": read_stats['partition rate'],
-            "row rate": read_stats['row rate'],
-            "latency 99th percentile": read_stats['latency 99th percentile'],
-        }
-        self.report_to_argus(ManagerReportType.READ, read_stress_report, "Read stress: " + label)
-
-        write_stress_report = {
-            "read time": int(stress_timer.duration.total_seconds()),
-            "op rate": write_stats['op rate'],
-            "partition rate": write_stats['partition rate'],
-            "row rate": write_stats['row rate'],
-            "latency 99th percentile": write_stats['latency 99th percentile'],
-        }
-        self.report_to_argus(ManagerReportType.READ, write_stress_report, "Write stress: " + label)
+    @latency_calculator_decorator
+    def mixed_latency_during_backup(self):
```

Comment:
It's not the method that should have workload type in name, rather test - need to rename `test_backup_benchmark` to `test_backup_benchmark_mixed`
or wait for: https://github.com/scylladb/scylla-cluster-tests/issues/9627